lib/everest/crc/include/everest/crc/crc.hpp

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 - 2025 Pionix GmbH and Contributors to EVerest
+
+#ifndef CRC_HPP
+#define CRC_HPP
+#include <stdint.h>
```

Comment:
you use  `cstdint` in the cpp but `stdint.h` here, should be updated.
Not that using the `cstdint` implementation requires the std:: namespace for `uint16_t` etc (at least depending on implementation, so it's safest to use the `std::` prefix everywhere